python-fedmsg_meta_umb.spec

```diff
@@ -0,0 +1,71 @@
+%global srcname fedmsg_meta_umb
+
+Name:           python-%{srcname}
```

Comment:
Ugh, should this turn into `python-fedmsg-meta-umb` in rpm land?

Reply:
That seemed confusing, since the upstream project uses underscores, but I see that's the convention with the other fedmsg_meta rpms. I'll make that change.